test/avocado/run-tests

```diff
@@ -30,12 +30,12 @@ import glob
 # the directory structure breaks on the second run with .pyc files
 sys.dont_write_bytecode = True
 
-import testinfra
-import testvm
-
-topdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
This was an issue when we had symlinks to files. If the avocado tests pass without the realpath, then I think we should leave it out.